dot/state/epoch_test.go

```diff
@@ -401,12 +401,24 @@ func TestStoreAndFinalizeBabeNextEpochData(t *testing.T) {
 			err := epochState.blockState.db.Put(headerKey(tt.finalizeHash), []byte{})
 			require.NoError(t, err)
 
+			// before finalize next epoch data this epoch should not be defined
```

Comment:
nit I think `for` is missing here?

```suggestion
			// before finalize next epoch data for this epoch should not be defined
```